examples/demo_quasardb.py

```diff
@@ -0,0 +1,33 @@
+from cryptofeed import FeedHandler
+from cryptofeed.exchanges import *
+from cryptofeed.backends.quasardb import *
+
+
+async def feed_info(data, receipt_timestamp):
+    print(f'{data} recived at {receipt_timestamp}')
+
+
+def main():
+    f = FeedHandler()
+
+    # save to database
+    f.add_feed(Coinbase(channels=[TICKER], symbols=['BTC-USD'], callbacks={TICKER: TickerQuasar()}))
```

Comment:
with the latest PR, coinbase no long has a ticker feed

Reply:
done, replaced with Binance